pkg/config/interface.go

```diff
@@ -27,6 +27,14 @@ type Config interface {
 	// GetVersion returns the current project version.
 	GetVersion() Version
 
+	// GetKubebuilderVersion returns the kubebuilder version used to initialize the project.
+	// This method was introduced in project version 3.
+	GetKubebuilderVersion() string
+
+	// SetKubebuilderVersion sets the kubebuilder version used to initialize the project.
+	//This method was introduced in project version 3.
+	SetKubebuilderVersion(version string) error
```

Comment:
Since others projects can import Kubebuilder and use it as lib to do their own CLI and things on top of Kubebuilder scaffolds. (i.e Operator-SDK) could we use here a generic term like GetCliVersion